web/src/components/Callout.tsx

```diff
@@ -39,35 +38,59 @@ export const calloutComponentMapping: Record<string, CalloutMappingObj> = {
   },
 };
 
-export const Callout = (props: Props): ReactElement => {
+export const Callout = (props: PropsWithChildren<Props>): ReactNode => {
   const { Config } = useConfig();
+  const showIcon = Boolean(props.icon);
   let headingText = "";
 
-  if (props.calloutType === "informational")
-    headingText = Config.calloutDefaults.informationalHeadingDefaultText;
-  if (props.calloutType === "note") headingText = Config.calloutDefaults.noteHeadingDefaultText;
-  if (props.calloutType === "conditional") headingText = Config.calloutDefaults.conditionalHeadingDefaultText;
-  if (props.calloutType === "warning") headingText = Config.calloutDefaults.warningHeadingDefaultText;
+  if (typeof props.header === "string" || props.icon === true || props.icon === "true") {
```

Comment:
So it seems like the .md content assumes that if props.header === undefined that we would default to `**Once completed, you will have:**`

<img width="1264" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/463e5e41-d90d-4973-9e25-2b7e70c9567b">

But because typeof props.header won't equal string if its undefined you don't get that default header

<img width="1845" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/3106a0e1-19d8-4523-95cd-079074719e9b">

Reply:
That or you're missing a lot of header=""'s in many places